plugins/org.jkiss.dbeaver.model.lsm/src/org/jkiss/dbeaver/model/stm/STMUtils.java

```diff
@@ -92,4 +90,66 @@ public static List<String> expandTermStrings(@NotNull STMTreeNode root) {
     }
 
 
+    public static <T, K> int binarySearchByKey(
```

Comment:
Javadocs :(

Reply:
I don't think, it's needed here. The name of the method is self-descriptive